[READY FOR MERGE] [VSEE-1244] [VSEE-1300] App-Scanning Supply Chain docs (#3292)
pivotal/docs-tap: scst-scan/getting-started.hbs.md

You can get started with Supply Chain Security Tools (SCST) - Scan 2.0 by using the following topics:

1. [Installing SCST - Scan 2.0](./install-app-scanning.hbs.md)

> 2. Create Supply Chain with App Scanning with either Supply Chain Choreographer for Tanzu or Tanzu Supply Chain:

Reviewer: ```suggestion
2. Create Supply Chain with SCST - Scan 2.0 with either Supply Chain Choreographer for Tanzu or Tanzu Supply Chain:
```

* Supply Chain Choreographer for Tanzu:

>   * [Add App Scanning to Default Test and Scan Supply Chains](./integrate-app-scanning.hbs.md)

Reviewer: ```suggestion
  * [Add SCST - Scan 2.0 to Default Test and Scan Supply Chains](./integrate-app-scanning.hbs.md)
```

  * [Bring your own scanner with Supply Chain Security Tools - Scan 2.0](./bring-your-own-scanner.hbs.md)
  * [Verifying Scanning with Supply Chain Integration](./verify-app-scanning-supply-chain.hbs.md)

* Tanzu Supply Chain:

>   * [Setup Supply Chain Component](./setup-supply-chain-component.md)

Reviewer: ```suggestion
  * [Setup the Supply Chain Component](./setup-supply-chain-component.md)
```

>   * [Create Supply Chain that uses App Scanning with a Component](./tanzu-supply-chain-with-app-scanning.md)

Reviewer: ```suggestion
  * [Create a Supply Chain that uses SCST - Scan 2.0 with a Component](./tanzu-supply-chain-with-app-scanning.md)
```

>   * [Create Workload from SupplyChain](./create-supply-chain-workload.md)

Reviewer: ```suggestion
  * [Create a Workload from the SupplyChain](./create-supply-chain-workload.md)
```
